integration/README.md

```diff
@@ -38,3 +72,13 @@ You can also run the CI tests and log collection as follows:
 ```bash
 ./scripts/integration.sh stop
 ```
+
+### Stop - Red Hat
+
+**Note** `stop` currently has an issue in this environment where you need to
+run it twice to fully clean up.
+
+```bash
+sudo ./scripts/integration.sh stop
```

Comment:
let's open an Issue on this

Reply:
Agree.  Was just waiting to get this merged before I did.  I think it's a matter of defining cross service dependencies so that services get stopped in the correct order.